cli/src/parser.sk

```diff
@@ -37,61 +37,71 @@ class ParseResults{
   subcommand: ?String = None(),
   error: ?ArgumentError = None(),
 } {
-  fun getBool(name: String): Bool {
-    this.values[name] match {
+  fun getBool(name: String, fail_on_invalid_arg: Bool = true): Bool {
+    this.getValue(name, fail_on_invalid_arg) match {
     | BoolValue(v) -> v
+    | MissingValue() -> false
```

Comment:
I haven't looked at how this interacts with the rest of `cli` but I suspect you may not always want `false` as the default. Should `getBool` take an optional `default: Option<Bool> = None()` instead of `fail_...`?

Reply:
`getBool` is the only one for which you have a default

Reply:
Oh, I see that `BoolArg` already defaults its default value to `false`...